src/opm/parser/eclipse/EclipseState/Schedule/Well/WellConnections.cpp

```diff
@@ -524,6 +562,32 @@ inline std::array< size_t, 3> directionIndices(const Opm::Connection::Direction
         }
     }
 
+    void WellConnections::excludeFromWellPI(const std::size_t connID) {
+        assert (!this->m_connections.empty() &&
```

Comment:
Use `throw std::lgoic_error()` instead of `assert()`

Reply:
> Use `throw std::logic_error()` 

Not needed in latest update, but I actually wanted `assert` here.  It was a programmer error to call this function if the connection set was empty and that check should not be there in release mode.

Reply:
Yes - but funny thing is programmer errors sometimes sneak through to release mode to?! At least in situations where the test is cheap I do not really understand the case for `assert()`?